### PR TITLE
Book library project

### PR DESCRIPTION
**Pull request by redconsole**
trying to raise PR only on behalf of Neeraj

### CODE REVIEW

File: src/main/java/com/education/demoeducation/ImpBook.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Iterator;
+import java.util.List;
+import java.util.ListIterator;
+
+@RestController
+@RequestMapping("/impBook")
```
Comment (redconsole) [inline src/main/java/com/education/demoeducation/ImpBook.java:11]: pls provide proper url for request mapping

Comment (redconsole) [reply src/main/java/com/education/demoeducation/ImpBook.java:11]: its for now, will change in next pr

File: src/main/java/com/education/demoeducation/ImpBook.java [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        return book1;
+    }*/
+    @PostMapping("/add1")
+    public void addBook(@RequestBody List<Book> book) {
+        library.books.addAll(book);
+   }
+
+    @PutMapping("/update")
+    public String update(@RequestBody Book book) {
+        ListIterator<Book> iterator= library.books.listIterator();
```
Comment (redconsole) [inline src/main/java/com/education/demoeducation/ImpBook.java:50]: remove unused code

**Review by redconsole — COMMENTED** [review]
done

File: src/main/java/com/education/demoeducation/ImpBook.java (hunk 2 above)
Comment (redconsole) [reply src/main/java/com/education/demoeducation/ImpBook.java:50]: done

**Comment by nkredc** [comment]
done all changes

**Review by nkredc — APPROVED** [review]
yes its fine

**Review by neerajmtr — APPROVED** [review]

**Pull request merged**